example/integration_tests/components-ce/sample/BUILD.bazel

```diff
@@ -20,6 +20,9 @@ native_image(
     name = "main-native",
     main_class = "Main",
     deps = [":java"],
+    extra_args = [
+        "--language:js",
+    ],
```

Comment:
enabling the component in the `native_image`: the `components-ce` sample, which was on the `release/0.10.x` branch, has taken on this test.